### DISCUSSION TITLE
Debugger Display for System.Diagnostics.Stopwatch

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/runtime#65722] [Ideas] [answered]
Could someone add a DebuggerDisplay to System.Diagnostics.Stopwatch please?

I think it should show something like the following:
```
00:00:25.6170218 (IsRunning = false)
```

This would save me countless clicks of having to expand object in Visual Studio.

**Comment by stephentoub** [comment, marked as answer]
This should be an issue / feature request rather than a discussion.
